### PR TITLE
feat(AMBER-664): Artwork page parity for private artworks

### PR DESCRIPTION
**Pull request by jpotts244**
This PR resolves [AMBER-665](https://artsyproduct.atlassian.net/browse/AMBER-665)

### Description

This PR brings the mobile artwork page to parity with web and the PA designs. I went over everything with Peter and he confirmed he is happy with the page. I also fixed some small bugs (Read more truncating wrong, price field rendering twice).

Screenshot 2024-05-22 at 1 58 12 PM
Screenshot 2024-05-22 at 1 58 05 PM
Screenshot 2024-05-22 at 1 56 51 PM

### To the reviewers 👀

- [X] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- brings mobile artwork page up to parity with web and private artworks design - lily

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[AMBER-665]: https://artsyproduct.atlassian.net/browse/AMBER-665?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Artwork/Artwork.tsx [hunk 1]
```diff
@@ -326,13 +336,6 @@ export const Artwork: React.FC<ArtworkProps> = (props) => {
         const hasEditionSets = (artworkAboveTheFold.editionSets ?? []).length > 1
 
         if (!!(artworkBelowTheFold?.isForSale && !isInAuction)) {
-          sections.push({
```
Comment (jpotts244) [inline src/app/Scenes/Artwork/Artwork.tsx:329]: Removes price as price is listed on the footer
Before:

Screenshot 2024-05-03 at 1 12 48 PM

---

After:
Screenshot 2024-05-03 at 12 48 12 PM

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 784b46180c8d97957bddcde57318d98afc2c2a21
</p>

File: src/app/Scenes/Artwork/Components/ShippingAndTaxes.tsx [hunk 2]
```diff
@@ -49,6 +42,13 @@ const ShippingAndTaxes: React.FC<ShippingAndTaxesProps> = ({ artwork }) => {
           {priceIncludesTaxDisplay}
         </Text>
       )}
+
+      {!!taxInfo && (
+        <Text variant="sm" color="black60">
+          {taxInfo.displayText}{" "}
+          <LinkText onPress={handleLearnMorePress}>{taxInfo.moreInfo.displayText}</LinkText>
+        </Text>
+      )}
```
Comment (jpotts244) [inline src/app/Scenes/Artwork/Components/ShippingAndTaxes.tsx:51]: Displays shipping and taxes info for private artworks too

---
Before:

Screenshot 2024-05-03 at 1 28 08 PM


---

After:
Screenshot 2024-05-03 at 12 50 39 PM

Comment (jpotts244) [reply src/app/Scenes/Artwork/Components/ShippingAndTaxes.tsx:51]: TODO: we want to track a tap event when `learn more` is tapped

Comment (jpotts244) [reply src/app/Scenes/Artwork/Components/ShippingAndTaxes.tsx]: Removing this commit to pull into a separate tracking work PR!

**Review by damassi — APPROVED** [review]
Looks good @jpotts244! So we did decide to go with all new components 👍

**Comment by jpotts244** [comment]
Closing as @lilyfromseattle is the true author here!

**Pull request closed without merging**